templates/fields.html.twig

```diff
@@ -65,7 +65,7 @@
     {% elseif type == 'url' %}
             {% set ext_link %}
             {% if value|length %}
-                <a target="_blank" href="{{ value }}">
+                <a target="_blank" href="{{ value|raw }}">
```

Comment:
`verbatim_value` is the correct filter to retrieve the initial value from user input. It decodes the contents, but is then escaped by Twig whereas `raw` filter prevents escaping.
If you set `http://www.test.com/index.php?a=2&truc=57" onclick="alert(1); return false;` as default value, with the `raw` filter, tou will be able to perform a XSS.

```suggestion
                <a target="_blank" href="{{ value|verbatim_value }}">
```